spec/presenters/notes_presenter_spec.rb

```diff
@@ -61,5 +61,19 @@
         expect(result[day2.beginning_of_day][0].body).to eq "Client added 4 documents."
       end
     end
+
+    context "with outbound calls" do
+      let(:day1) { DateTime.new(2019, 10, 5, 8, 1).utc }
+      let(:day2) { DateTime.new(2020, 10, 5, 5, 1).utc }
+      let!(:outbound_call_completed) { create :outbound_call, twilio_status: "completed", created_at: day1, call_duration: "75", client: client, user: user, note: "I talked to them!" }
+      let!(:outbound_call_queued) { create :outbound_call, twilio_status: "queued", created_at: day2, client: client, user: user }
+      it "does not include outbound_calls in status queued" do
+        result = NotesPresenter.grouped_notes(client)
+        notes = result.values.flatten
+        expect(notes.length).to eq 1
+        expect(notes[0].body).to include "Called by Gary Gnome. Call was completed and lasted 1m15s."
+        expect(notes[0].body).to include "I talked to them!"
```

Comment:
Consider keeping the `to eq 1` here, but moving the text assertions to another test.